hal/include/HAL/Accelerometer.h

```diff
@@ -9,11 +9,14 @@
 
 #include "HAL/Types.h"
 
-enum HAL_AccelerometerRange : int32_t {
+/* clang-format off */
```

Comment:
If the goal of this commit is to make it ANSI C, C89, then no line comments. If we are using C99, which windows doesn't fully support, then I'll switch it.